app/components/QuoteForm.tsx

```diff
@@ -15,6 +22,14 @@ interface QuoteFormProps {
   field2Placeholder?: string;
   field3Placeholder?: string;
   submitButtonText?: string;
+  setSubmitSuccessful: Dispatch<SetStateAction<boolean>>;
+}
+
+export interface FormData {
+  name: string;
+  email: string;
+  industry: string;
+  details: string;
```

Comment:
Considering that industry and details fields are currently not required, I suggest making these properties optional unless I'm missing something else
```suggestion
  industry?: string;
  details?: string;
```